os/tools/mksyscall.c

```diff
@@ -224,6 +224,8 @@ static void generate_proxy(int nparms)
 	int nactual;
 	char *cond_parm;
 	int i;
+	bool specialcon =false;
```

Comment:
Let us change variable to `special_case`.
Take care of coding rule. Space before `false`